pgjdbc/build.gradle.kts

```diff
@@ -224,6 +224,8 @@ tasks.configureEach<Jar> {
 tasks.shadowJar {
     configurations = listOf(shaded)
     exclude("META-INF/maven/**")
+    exclude("META-INF/versions/**")
```

Comment:
What is the reason for excluding `/versions/**`?

Reply:
Right now it only has the module-info descriptor, and when a dependency is shaded, the descriptor is useless (and broken).

Reply:
Good to know, it would be great to add that as a code comment. Should we rather exclude a specific `/versions/9/module-info.class`?

Reply:
Yes, probably is better to be specific, BTW, apparently this exclude does not work with specific files, that's why there is a TODO in the service file.

Also, I lack experience with Gradle, so if you can provide the fix for this would appreciate.

Reply:
Looks like specific excludes works.